### ISSUE TITLE
Allow for Esc key to close any dialogs / modal currently in focus

### ISSUE THREAD

**Issue by sergioc** [Particular/ServiceControlInstaller#37]
Check that these Screens behave correctly when ESC key pressed 

- [x] Add Instance Screen Closes on Esc
- [x] Edit Instance Screen Closes on Esc
- [x] Delete Instance Screen Closes on Esc
- [x] Unhandled Exception Screen Closes on Esc
- [x] Online Feedback dialog Closes on Esc
- [x] Offline Feedback dialog Closes on Esc
- [x] Upgrade dialog Closes on Esc (MessageBox)
- [x] License dialog Closes on Esc

Basically, any SC Management dialog should Close/Cancel on Esc.

**Comment by petersgiles** [comment]
@gbiellem @sergioc are there any other screens that require this?

**Comment by sergioc** [comment]
@petersgiles I added more to the list above.

**Comment by petersgiles** [comment]
Reopened - The Key Press is bubbling up to the window on dialogs closing everything
